packages/headless/src/controllers/core/triggers/headless-core-redirection-trigger.ts

```diff
@@ -0,0 +1,21 @@
+import {Controller} from '../../controller/headless-controller';
```

Comment:
We could potentially extract an actual "core" controller, but this is something I'd rather do once https://github.com/coveo/ui-kit/pull/3890 is merged.